src/openrct2/rct1/RCT1.h

```diff
@@ -64,7 +64,7 @@ struct rct1_ride
     colour_t track_primary_colour;                            // 0x01E
     colour_t track_secondary_colour;                          // 0x01F
     colour_t track_support_colour;                            // 0x020
-    uint8_t status;                                           // 0x021
+    RideStatus status;                                        // 0x021
```

Comment:
In general the header files in rct1 and rct2 are left without the enum classes. This is because the field is meant to be verified that it is within bounds before inporting into the game.